### PR TITLE
Km 81 connect gcloud db prisma

### PR DESCRIPTION
**Pull request by Maeaex**
Hinzufügen .env in Prisma Folder mit Connection String (siehe Email mit .env File an euch). 

Der Pull Request ist basierend auf Alex's Branch "KM-91 -POC Prisma". 

Alex Migration in die DB hat über Prisma tadellos geklappt.

image

### CODE REVIEW

**Review by beckertobias — APPROVED** [review]
Looks good to me. I didn't test it but I trust you that it's working.
I already had a look at Alex' PR before.
We just have to pay attention: If we merge this PR, we shouldn't merge Alex' PR because this one already includes his commits.

File: .gitignore [hunk 1]
```diff
@@ -37,3 +38,10 @@ yarn-error.log*
 
 # test db
 
+#google cloud
+mythical-module-308515-8f036fb55006.json
+app.yaml
+
+#prisma
+prisma/.env
```
Comment (atsteinberg) [inline .gitignore:46]: there is no such file. What is this supposed to achieve?

File: .gitignore [hunk 2]
```diff
@@ -37,3 +38,10 @@ yarn-error.log*
 
 # test db
 
+#google cloud
+mythical-module-308515-8f036fb55006.json
+app.yaml
```
Comment (atsteinberg) [inline .gitignore:43]: there is no such file. What is this supposed to achieve?

File: .gitignore [hunk 3]
```diff
@@ -37,3 +38,10 @@ yarn-error.log*
 
 # test db
 
+#google cloud
+mythical-module-308515-8f036fb55006.json
```
Comment (atsteinberg) [inline .gitignore:42]: there is no such file. What is this supposed to achieve?

File: apollo/helpers.ts [hunk 4]
```diff
@@ -38,14 +39,21 @@ const getExperienceById = (
...
-  models.Talent.findMany().map((talent: User) => talent.id);
+export const getAllTalentIds = async (): Promise<string[] | undefined> => {
+  try {
+    const talents = await prisma.talent.findMany();
+    return talents.map((talent) => talent.id.toString()) || ['none'];
+  } catch (e) {
+    handleError(e);
+  }
+};
+// return models.Talent.findMany().map((talent: User) => talent.id);
```
Comment (atsteinberg) [inline apollo/helpers.ts:50]: please delete comment

File: pages/talents/[id]/index.tsx [hunk 5]
```diff
@@ -39,6 +39,12 @@ import {
 import { getShortName } from '../../../lib/utils/strings';
 import useStyles from './profile.styles';
 
+const GET_ALL_TALENTS = gql`
```
Comment (atsteinberg) [inline pages/talents/[id]/index.tsx:42]: please remove code that is for testing purposes before merging to main

File: pages/talents/[id]/index.tsx [hunk 6]
```diff
@@ -134,6 +140,7 @@ const GET_ALL_INFO = gql`
 `;
 
 const ProfilePage = ({ t, i18n }: PageProps): React.ReactElement => {
+  const { data: talentIds, loading: idLoading } = useQuery(GET_ALL_TALENTS);
```
Comment (atsteinberg) [inline pages/talents/[id]/index.tsx:143]: please remove code that is for testing purposes before merging to main

File: pages/talents/[id]/index.tsx [hunk 7]
```diff
@@ -144,6 +151,8 @@ const ProfilePage = ({ t, i18n }: PageProps): React.ReactElement => {
   const [modal, setModal] = useState<{ type: ModalType; id?: string }>({
     type: ModalType.NONE,
   });
+  if (idLoading) return <></>;
+  console.log(talentIds);
```
Comment (atsteinberg) [inline pages/talents/[id]/index.tsx:155]: please remove code that is for testing purposes before merging to main

File: pages/talents/[id]/index.tsx [hunk 8]
```diff
@@ -291,29 +300,4 @@ const ProfilePage = ({ t, i18n }: PageProps): React.ReactElement => {
...
-
-// export const getStaticProps: GetStaticProps = async ({ params }) => {
-//   return {
-//     props: {
-//       id: params?.id,
-//     },
-//   };
-// };
-
-ProfilePage.getInitialProps = async () => ({
```
Comment (atsteinberg) [inline pages/talents/[id]/index.tsx:315]: we might need this. @beckertobias ?

**Review by atsteinberg — CHANGES_REQUESTED** [review]
Hey Max. Sorry, this would have been easier without KM-91 mixed in. So, could you just quickly say which changes you made (or even better: if it was not a lot: do them again in a clean branch from current main)? That would be awesome!

File: .gitignore (hunk 3 above)
Comment (atsteinberg) [reply .gitignore:42]: or is this generated once you run something?

File: .gitignore (hunk 1 above)
Comment (atsteinberg) [inline .gitignore:46]: this is already caught by line 30

**Comment by atsteinberg** [comment]
I'm closing this mr, because it is already included in merged KM-91.

**Pull request closed without merging**